### DISCUSSION TITLE
Help on converting custom data to suitable tsai format

### DISCUSSION THREAD

**Discussion by Zwayeh** [timeseriesAI/tsai#644] [General]
Hi! 

Absolutely love the concept of this work! Time series really seems to be a growing research area and tools like these make is significantly easier for a beginner like myself to have a go. :)

Apologies in advance as I imagine this is quite a simplistic solution but I have been unable to get it to work correctly so I am coming here for some help. I am looking to test ROCKET (and eventually Mini-ROCKET) on the PHAC-2 dataset. I have converted this dataset to .csv format (attached). Each row indicates a reading, with Column A indicating Class while the remainder of columns are the actual data - structure is similar to UCR OliveOil.
[TACTEST.csv](https://github.com/timeseriesAI/tsai/files/10213666/TACTEST.csv)

I was wondering what steps I need to take to convert this to a suitable structure and/or data type for this to be usable in the 02_ROCKET_a_new_SOTA_classifier workbook. I have looked at the 00c_Time_Series_data_preparation notebook, and have converted the .csv to a pandas dataframe, but following this I get a little lost on what my next steps should be.

Thanks in advance and apologies again if this has been answered elsewhere!

**Comment by Zwayeh** [comment]
Issue resolved :)

**Reply by kns97** [reply]
Hello how do we upload downloaded data to tsai framework? I only see options to download from UCR dataset or a URL in 00c_Time_Series_data_preparation notebook. Thank you in advance,

**Reply by oguiza** [reply]
Hi @kns97 ,
I'm sorry but that's a very general question. The UCR datasets are used to demonstrate what the input to tsai needs to look like. But the process you follow will depend on the type of data that you have.
